src/features/languages/generated/client/client.json

```diff
@@ -1431,7 +1440,7 @@
         "recordAudit.noInformant": "No informant",
         "recordAudit.noName": "No name provided",
         "recordAudit.history.started":"Started",
-        "recordAudit.history.reinstated.prefix":"Reinstated to ",
+        "recordAudit.history.reinstated":"Reinstated to {status}",
```

Comment:
Whats wrong with the key?

Reply:
The key's been changed in the core PR as well as we don't really need the _prefix_ part here